sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,217 @@
+- Signers must deduct transaction fees from users in order to fund BTC withdrawal transactions.
+    - They must ensure that the transaction fee is paid for (e.g., they deduct it from the user, and they set a minimum sBTC withdrawal amount).
+    - The process for handling transaction fee estimates is described in this [issue](https://github.com/stacks-network/sbtc/pull/186).
+- Signers must coordinate to set and advertise the fee parameters of the system.
+    - They must decide a minimum sBTC peg-out.
+    - They must decide an STX transaction fee for minting the sBTC. This fee is paid by the user and can be sponsored by a 3rd party, which is described in the “Auxiliary Features” section of this document.
+
+### sBTC Signer Eligibility Criteria
+
+Signers will run the sBTC binary in addition to the core Stacks signer software and must meet certain criteria in order to facilitate the reliable functioning of the sBTC protocol at all times.
+
+The following eligibility criteria will be used to identify the sBTC Signers:
+
+- **Company Standing:**  
+  Does the company have an operating history to demonstrate their experience and reliability?
+- **Stacks Participation:**  
+  Has the Signer participated in running a signer node on Stacks 2.5 testnet or mainnet?
+- **Network Uptime:**  
+  Does the Signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer?  
+  *Note: This metric will be self-affirmed by the company.*
+- **Communication & Availability:**  
+  Does the signer have a direct communication channel set up with sBTC core engineers to be able to respond to urgent updates within 24 hours? (e.g., Slack, Telegram, Signal)
+- **Ecosystem Alignment:**  
+  Has the signer made contributions to Bitcoin or the Stacks network over the past year that demonstrate their commitment to the growth and success of the network? Examples include, but are not limited to: publishing independent research, marketing, co-authoring a SIP, submitting a Stacks pull request, providing feedback on Stacks core development, or contributing to a Stacks working group.
+
+The criteria described above will be used to identify sBTC Signers that are able to meet some or all of the responsibilities described in the previous section.
+
+### Selection Process
+
+The sBTC Signer Set will be finalized from the list of eligible Signers, based on the above criteria, by the [sBTC working group](https://github.com/orgs/stacks-network/discussions/469). This process is to ensure that the sBTC Working Group can adjust to any changes in the sBTC Signer Set quickly and without the overhead of an additional community vote. For example, if an sBTC Signer is no longer available to complete their responsibilities, it is imperative for the liveness of the system that the sBTC Working Group is able to implement a suitable replacement based on the above criteria.
```

Comment:
> This process is to ensure that the sBTC Working Group can adjust to any changes in the sBTC Signer Set quickly and without the overhead of an additional community vote.

I think this may be a case of you "saying the quiet part out loud."  I can understand the desire to avoid the time costs of a long-running vote, but surely there must be some means for community oversight of the signer set?

Reply:
I think once you have an initial signing set, then you can have future SIPs for changing it.  Unlike this SIP, I think there's room for SIPs that only change the signing set, which can have shorter voting windows and only require participation from sBTC holders (since they'd be the only stakeholders).

Reply:
i believe this is addressed by 09944a5ecc79ca0852b3ef2cea9e2b7c407d1644